kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtNetworkServiceImplFacade.java

```diff
@@ -144,14 +144,14 @@ public boolean isIEEE80211ACSupported(GwtXSRFToken xsrfToken, String ifaceName)
     }
 
     @Override
-    public List<String> getDhcpLeases(GwtXSRFToken xsrfToken) throws GwtKuraException {
+    public List<String> getDhcpLeases(GwtXSRFToken xsrfToken, String interfaceName) throws GwtKuraException {
         checkXSRFToken(xsrfToken);
 
         if (isNet2()) {
             // TODO
             return new ArrayList<>();
         } else {
-            return org.eclipse.kura.web.server.net.GwtNetworkServiceImpl.getDhcpLeases();
+            return org.eclipse.kura.web.server.net.GwtNetworkServiceImpl.getDhcpLeases(interfaceName);
```

Comment:
Maybe we can call `org.eclipse.kura.web.server.net.GwtNetworkServiceImpl.getDhcpLeases(interfaceName);` also for net2 case.

Reply:
We can't since that method uses the `NetworkAdminService` that is not available on net2.